src/components/AddStudent/AddStudentForm.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React from "react";
+import {Field, reduxForm} from "redux-form";
+import RaisedButton from "material-ui/RaisedButton";
+import {renderTextField, renderDatePicker, required, minLength2, alphaNum} from "../../utils/ReduxFormUtils";
+import ValidateButton from "../ValidateButton/ValidateButton";
+
+/**
+ * Formulaire d'ajout d'élève.
+ * @param props
+ * @returns {XML}
+ * @constructor
+ */
+
+
+
+let AddStudentForm = props => {
+    /**
+     * handleSubmit => fonction de gestion du formulaire (correspond au onSubmit donné lors de l'instanciation)
+     * invalid => boolean à faux si le formulaire est invalide. Vrai sinon.
+     * onCancel => fonction donné lors de l'instanciation du composant. Appelé lors d'un clique sur le bouton annuler.
+     */
+    const {valid, onCancel, handleSubmit, isLoading, formatDate} = props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "formatDate" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'formatDate' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)